src/walter/project.py

```diff
@@ -241,7 +241,11 @@ def run(self, sim_id=None, dry_run=False, **kwds):
         lsys, lstring = None, None
         if not dry_run:
             lsys = Lsystem(self.walter, {'params': kwds, 'ID': sim_id})
+            time_start = time.time()
             lstring = lsys.iterate()
+            time_stop = time.time()
+            with open(self.dirname/'output'/sim_id/"Simulation_time.txt", 'w') as time_file:
```

Comment:
Use the method output_path rather hard coding it in this function. This will prevent later bugs. You will just have one function to change rather than modifying all the functions.

If output_path does not correspond to what you want, extend it (for instance with sim_id)

Reply:
Thank you for your advice, I made the necessary changes. I think the pull-request is ready to be merged now, unless you have other suggestions.

Reply:
May be we can use this occasion instead to make output_path method, used without args, to point to the latest ('current') simulation that has run, which would be more intuitive than the current behavior (output_path without args points to the last sim_id added to itable, even if the last run has been done for another id) ? In this case, we will just have to call self.output_path().

Reply:
I understand how this would be better but I do not see how to do this change concretely. Partly because, when we launch several simulations, they run simultaneously as different subprocesses.